Runtimes/Overlay/Windows/clang/CMakeLists.txt

```diff
@@ -1,4 +1,46 @@
 
+file(TO_CMAKE_PATH "$ENV{WindowsSdkDir}" WindowsSdkDir)
+file(TO_CMAKE_PATH "$ENV{WindowsSDKVersion}" WindowsSDKVersion)
+file(TO_CMAKE_PATH "$ENV{UniversalCRTSdkDir}" UniversalCRTSdkDir)
+file(TO_CMAKE_PATH "$ENV{UCRTVersion}" UCRTVersion)
+file(TO_CMAKE_PATH "$ENV{VCToolsInstallDir}" VCToolsInstallDir)
```

Comment:
For my education: does it make sense to be able to set those through CMake variables as well, or can we always rely on these being set when building for Windows? I assume the latter from the quick searches I did [1] [2]

[1]: https://discourse.cmake.org/t/visual-studio-install-path-and-directory-query/3684
[2]: https://github.com/microsoft/vswhere/wiki/Start-Developer-Command-Prompt